### DISCUSSION TITLE
Password reset from admin panel

### DISCUSSION THREAD

**Discussion by palicepta** [dani-garcia/vaultwarden#3841] [Ideas]
Dear All,

Why we can't change the password for any user from the admin panel?

<img width="1060" alt="image" src="https://github.com/dani-garcia/vaultwarden/assets/112872766/cae469ba-8d9d-4f69-8690-64bfda9c2569">

Thank you.

**Comment by stefan0xC** [comment]
Because for [account recovery ](https://bitwarden.com/help/account-recovery/) to work, users need to be in an organization with the policy enabled and also actively enrolled in the policy (or auto-enrolled). Also I think we would have to be a member of the organization to be able to reset the password for an account with access to an organization (otherwise it would be a severe security flaw because everybody with `/admin` access could gain access to the encrypted organization data).

**Reply by BlackDex** [reply]
@stefan0xC is right. There are several keys needed to be able to reset a password of an other enrolled user.
Those keys can only be decrypted by an owner/admin who is a member of that same org.

**Comment by palicepta** [comment]
Dear [stefan0xC](https://github.com/stefan0xC)

For example situation: 
The user did not allow this policy before, and he forgot the password.

Now I should reset his password.

How I can do this?

**Reply by BlackDex** [reply]
You can't That is the whole point of a secure password manager.
The only way the user can gain access is by remembering his password again, or if he still has access to an unlocked client, they can export it from there if i'm correct.

Otherwise, they are in bad luck, and lost there passwords.

**Reply by palicepta** [reply]
[BlackDex](https://github.com/BlackDex) Thank you for your answer.

**Reply by BlackDex** [reply]
Also, a good option is to have people add an emergency contact which would help gaining access to there password via a trusted person.
